cypress/e2e/tests/administration/custom-migration-targets/crud.test.ts

```diff
@@ -165,7 +165,7 @@ describe(["@tier1", "@interop"], "Custom Migration Targets CRUD operations", ()
     });
 
     Object.values(Languages).forEach((language) => {
-        it(`${language} | Change layout and check in analysis wizard`, function () {
+        it(`Bug MTA-3308: ${language} | Change layout and check in analysis wizard`, function () {
```

Comment:
If the bug affects just the Go languaje then that test should be the only one marked
```typescript
 it(`${language === Languages.Go ? "Bug MTA-3308:" : "" } ${language} | Change layout and check in analysis wizard`, function () {
 ```

Reply:
Nice, I will apply that. Thanks